thoth/common/workflows_test.py

```diff
@@ -0,0 +1,46 @@
+"""Workflows test suite."""
```

Comment:
It would be a good idea to move the test suite out of the prod code. You can place them under `tests/` directory as in the case of other libraries in Thoth. Bots will run the test suite on each PR. I created the `tests/` dir in https://github.com/thoth-station/common/pull/529

Reply:
Done :) Thanks!